README.md

```diff
@@ -456,6 +457,7 @@ Or via environment variables:
 * `THEFUCK_RULES` &ndash; list of enabled rules, like `DEFAULT_RULES:rm_root` or `sudo:no_command`;
 * `THEFUCK_EXCLUDE_RULES` &ndash; list of disabled rules, like `git_pull:git_push`;
 * `THEFUCK_REQUIRE_CONFIRMATION` &ndash; require confirmation before running new command, `true/false`;
+* `THEFUCK_REQUIRE_DOUBLE_CONFIRMATION` &ndash; require double confirmation before running potentially volitile commands (eg. `reboot`), value of `true/false`;
```

Comment:
This really should have been a separate PR, since it makes it default now both sets of changes have to be approved before either can get merged.